src/js/stores/library.js

```diff
@@ -196,9 +199,25 @@ define(function (require, exports, module) {
             this._libraryCollection = payload.collection;
             this._updateLibraries(payload.lastSelectedLibraryID);
             
-            var editStatusArray = _.map(payload.editStatus, function (status, documentID) {
-                return [parseInt(documentID), status];
-            });
+            var editStatusArray = _.reduce(payload.editStatus, function (result, status, documentIDStr) {
+                // Remove orphan edit status that are not associate with any open documents.
+
+                var documentID = math.parseNumber(documentIDStr);
```

Comment:
@iwehrman `payload.editStatus ` is loaded from preference and it is stored as string using `JSON.stringify`, its keys are stringified to match JSON spec, that's why the documentID is string here. I gave it a better name, and use `math.parseNumber` as suggested to deal with edge cases.